### PR TITLE
Add more German translation

### PR DESCRIPTION
**Pull request by JohnnyLeone**

### CODE REVIEW

**Comment by akien-mga** [comment]
Sorry about the merge conflicts, I had to push f4ee46124a to ensure more consistency in the future. I'll cherry-pick your commit manually.

File: tools/translations/de.po [hunk 1]
```diff
@@ -404,15 +404,15 @@ msgstr ""
 
 #: tools/editor/project_export.cpp
 msgid "Shrink All Images:"
-msgstr ""
+msgstr "Alle Bilder verkleinern"
```
Comment (akien-mga) [inline tools/translations/de.po]: Be careful about the punctuation, it's often there for a reason :)

File: tools/translations/de.po [hunk 2]
```diff
@@ -536,31 +536,31 @@ msgstr ""
...
 
 #: tools/editor/project_export.cpp
 msgid "Project Export"
-msgstr ""
+msgstr "Projekt exportieren"
 
 #: tools/editor/project_export.cpp
 msgid "Export Preset:"
-msgstr ""
+msgstr "Vorlage exportieren"
```
Comment (akien-mga) [inline tools/translations/de.po]: For example here, by ignoring the punctuation you mistranslated the string. It should likely be "Exportvorlage:". I'll fix it as I'm merging the commit manually anyway, but keep that in mind for the next time :)

Comment (akien-mga) [inline tools/translations/de.po]: Actually it could be translated both ways, let me check :p

Comment (akien-mga) [inline tools/translations/de.po]: In looks like this code is not used in the export wizard, so I can't check :D Anyway, you get the point about the punctuation ;)

**Comment by akien-mga** [comment]
Merged as da25b76aa.

**Comment by JohnnyLeone** [comment]
Oh, alright sorry. I'll watch better in Future... Thanks!

**Pull request closed without merging**